zkevm-circuits/src/super_circuit.rs

```diff
@@ -122,7 +122,10 @@ impl<F: Field> SubCircuitConfig<F> for SuperCircuitConfig<F> {
         }: Self::ConfigArgs,
     ) -> Self {
         let tx_table = TxTable::construct(meta);
+
+        let chronological_rw_table = RwTable::construct(meta);
```

Comment:
I think it would be more clear if here we see the two rw_tables: `chronological_rw_table` and `by_address_memory_table`.
One is passed to the evm_circuit and the other to the state_circuit.

Reply:
fixed and renamed by by_address_rw_table